Client/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/queues/QueueConnector.java

```diff
@@ -0,0 +1,61 @@
+package pl.jakubowskiprzemyslaw.tajgertim.queues;
+
+import com.rabbitmq.client.Channel;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+import com.rabbitmq.client.GetResponse;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+@Component
+public class QueueConnector {
+
+    private Connection connection;
+    private Channel channel;
+
+    public void connectToQueue(String queueName) throws IOException, TimeoutException {
+        createConnection();
+        createChannel();
+        declareQueue(queueName);
+    }
+
+    private void createConnection() throws IOException, TimeoutException {
+        ConnectionFactory factory = new ConnectionFactory();
+        factory.setUsername("admin");
+        factory.setPassword("hidden");
+        factory.setHost("10.30.1.140");
+
+        connection = factory.newConnection();
+    }
+
+    private void createChannel() throws IOException {
+        channel = connection.createChannel();
+    }
+
+    private void declareQueue(String queueName) throws IOException {
+        boolean queueAutoDelete = false;
+        boolean queueExclusive = false;
+        boolean queueDurable = false;
+        channel.queueDeclare(queueName, queueDurable, queueExclusive, queueAutoDelete, null);
+    }
+
+    void sendMessage(String queueName, String json) throws IOException {
+        channel.basicPublish("", queueName, null, json.getBytes());
```

Comment:
Pusty string?

Reply:
Nie został nadany żaden tag kolejki, dlatego tak to wygląda